aries-site/src/pages/components/notification.mdx

```diff
@@ -0,0 +1,71 @@
+import { StateVsStatusDiagram, TaxonomyTable, ToastNotificationExample } from '../../examples';
+import { Example } from '../../layouts';
+import { Box, Text } from 'grommet';
+
+## What are they for?
+
+Notifications build trust by providing users confidence and assurance that they are informed with timely, relevant status about their systems and tasks.
+
+## Taxonomy 
+
+To standardize language for Notifications across HPE projects, be sure to familiarize yourself with this taxonomy:
+
+- **Type** - In the broadest terms, what the notification is about
+- **State** - The current stage within a process
+- **Status** - The condition of a process
+- **Attention Level** - The priority and importance of a notification
+- **Actions** - Interactactions within a notification
+- **Persistence** - How the notification is dismissed
+- **Placement** - Where the notification can be found
+- **Insight** - The content of the notification
+
+<TaxonomyTable />
+
+### State v.s. Status
+
+A state is the current stage of a process.
+
+A status is the condition of the state. Status can also be thought of as a pointer to a state’s “health”. 
+
+Here is an example of a Toast Notification that illustrates their differences:
+
+<StateVsStatusDiagram />
+
+Keep in mind that a status **derives** from a state.
+
+## Notification Systems
+
+The following types of notifications are used to provide users with full transparent clarity on tasks and system statuses:
+
+- [Status Indicators](/components/status-indicator) are used within all types of notifications. They capture a user’s attention based on the notification’s level of severity.
+- For low priority and non-action-oriented notifications, [Toast Notifications](/templates/toast-notifications) are a great way to inform users unobtrusively. 
+
+## Variants
+
+There are a couple "types" of the Notification component depending on use case.
+
+### Toast Notification
+
+Toast notifications are used for low attention status messages or updates that do not require user action. For more thorough guidance on how and when to properly make use of a toast, access the full page on [Toast Notifications](/templates/toast-notifications).
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/toast-notifications/ToastNotificationExample.js"
```

Comment:
If we update the folder where the examples live, we should also make sure to update the code links